core/src/core-api/featuretoggles.js

```diff
@@ -0,0 +1,60 @@
+import { get, writable } from 'svelte/store';
+/**
+ * Functions to use feature toggles in Luigi
+ * @name FeatureToggles
+ */
+class LuigiFeatureToggles {
+  constructor() {
+    this.featureToggleList = writable([]);
+  }
+  /**
+   * Add a feature toggle to an active feature toggles list
+   * @memberof FeatureToggles
+   * @since NEXTRELEASE
+   * @example Luigi.featureToggles().setFeatureToggle('featureToggleName');
+   */
+  setFeatureToggle(featureToggleName) {
+    if (featureToggleName && typeof featureToggleName === 'string') {
+      if (!get(this.featureToggleList).includes(featureToggleName)) {
+        get(this.featureToggleList).push(featureToggleName);
+      } else {
+        console.log('Feature toggle name already exists');
+      }
+    } else {
+      console.log('Feature toggle name is empty or not a type of string');
+    }
+  }
+
+  /**
+   * Remove a feature toggle from the list
+   * @memberof FeatureToggles
+   * @since NEXTRELEASE
+   * @example Luigi.featureToggles().unsetFeatureToggle('featureToggleName');
+   */
+  unsetFeatureToggle(featureToggleName) {
+    if (featureToggleName && typeof featureToggleName === 'string') {
+      if (get(this.featureToggleList).includes(featureToggleName)) {
+        let index = get(this.featureToggleList).indexOf(featureToggleName);
+        if (index > -1) {
+          get(this.featureToggleList).splice(index, 1);
+        }
+      } else {
+        console.log('Feature toggle name is not in the list.');
+      }
+    } else {
+      console.log('Feature toggle name is empty or not a type of string');
+    }
+  }
+
+  /**
+   * Get a list of active feature toggles
+   * @memberof FeatureToggles
+   * @since NEXTRELEASE
+   * @return {Array} of active feature toggles
+   * @example Luigi.featureToggles().getActiveFeatureToggleList();
+   */
+  getActiveFeatureToggleList() {
+    return get(this.featureToggleList);
```

Comment:
Would be awesome to avoid possible mutations by returning a clone instead of reference.